### PR TITLE
Syndicate Ship

### PR DESCRIPTION
**Pull request by dapocalypse**
### Purpose
Adds a syndacate ship with gear and moves the spawn there.

### Approach
Build it, adds items where they can pick up.

### Open Questions and Pre-Merge TODOs

- [ +]  I read the contribution guidelines and the wiki.
- [ +]  This fix is tested on the same branch it is PR'ed to.
- [Everything was done in the map editor]  I correctly commented my code
- [ +]  This PR does not include files without specific need to do so.
- [ +]  This PR does not bring up any new compile errors
- [ +]  This PR has been tested in editor
- [ Dont know how to test]  This PR has been tested in multiplayer

### Notes:
dedhiadev5@gmail.com
#974

### CODE REVIEW

**Comment by fomalsd** [comment]
Not knowing how to test doesn't mean you shouldn't test, here's a guide: 
https://github.com/unitystation/unitystation/wiki/Building-And-Testing

File: UnityProject/Assets/Materials/Resources/OverlayCritHole.mat [hunk 1]
```diff
@@ -23,8 +23,8 @@ Material:
         m_Scale: {x: 1, y: 1}
         m_Offset: {x: 0, y: 0}
     m_Floats:
-    - _Radius: 0.03
-    - _Shape: 2
+    - _Radius: 0.38
```
Comment (fomalsd) [inline UnityProject/Assets/Materials/Resources/OverlayCritHole.mat]: This file doesn't need to be included

File: UnityProject/Assets/Tilemaps/Palettes/Windows.prefab [hunk 2]
```diff
@@ -149,7 +149,7 @@ Tilemap:
       m_TileColorIndex: 1
       m_ObjectToInstantiate: {fileID: 0}
       m_TileFlags: 0
-      m_ColliderType: 0
```
Comment (fomalsd) [inline UnityProject/Assets/Tilemaps/Palettes/Windows.prefab:152]: This file also doesn't need to be included

**Comment by dapocalypse** [comment]
Edit multiplayer is good

**Review by fomalsd — CHANGES_REQUESTED** [review]
Critera 1 and 2 not met: Nuke Ops ship is on the station matrix and not the separate one. Refer to how the other two ships are mapped. At the moment the ship is just a part of the station, and mapped so close that even station's catwalk sticks to it.
Also there are two unnecessary files included in the PR.
Finally, is it meant to be not symmetrical?

**Comment by dapocalypse** [comment]
It's not ment to be symetrical

**Pull request closed without merging**